css-three-column-layout/styles.css

```diff
@@ -0,0 +1,62 @@
+* {
+  background-color: #4895ef;
+}
+
+h2,
+h3,
+p {
+  color: white;
+  font-family: 'Lato', sans-serif;
+}
+
+h3 {
+  height: 0;
+}
+
+.gray {
+  color: lightgray;
+  margin: 15px;
+  font-size: small;
+  line-height: 140%;
+}
+
+.heading {
+  width: 100%;
+  text-align: center;
+}
+
+.container {
+  max-width: 800px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.row {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  padding: 8px;
```

Comment:
The only styling applied to a `row` class should be `display: flex`, you should make the rest of this styling a separate class you can add in addition to `row`